apps/dashboard/src/app/App.tsx

```diff
@@ -121,6 +122,11 @@ export class App extends React.Component<IAppSetting, IAppState> {
                   this.state.version
                 ]
               }
+              featureFlights={
+                this.state.featureFlights === "none"
```

Comment:
It can't be because we're not allowing it to be. I can look into whether I can change that, but I'd have to change it for all the other app settings as well.

Reply:
Here's the problem: we generate the URL for the test env based on the input parameters. If we allow empty args like this then it fails in 
```
export function generateRoute(params: readonly string[]): string {
  return params.map((p) => `/:${p}?`).join("");
}
```
because we're expecting at least one character.

That's why I introduced "none" as a special case (in the test env only, this doesn't exist in the actual app). Even if we were to figure out how to allow for empty (or undefined) flight params we'd still need to handle this case when showing the currently selected flights in the multi-select dropdown. Right now it can say "none" and it's intuitive. Otherwise we'd have to handle this in a different way. Anyway, I guess this is my argument for keeping it the way it is 🙂

Reply:
So even empty does not work, does it?

Reply:
No because `generateRoute` doesn't allow that.

Reply:
It's this particular route generation that won't work because you need characters:
![image](https://user-images.githubusercontent.com/10245648/167307626-c20011aa-fa54-41e9-8ad0-9261d5c88770.png)

Hence "none". Only in the test env, though, and it's just for the special case where we have no flights. Otherwise it uses the flight names.